benchmark/zlib/pipe.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const common = require('../common.js');
+const fs = require('fs');
+const zlib = require('zlib');
+
+const bench = common.createBenchmark(main, {
+  inputLen: [1024],
+  duration: [5],
+  type: ['string', 'buffer']
+});
+
+function main({ inputLen, duration, type }) {
+  const buffer = Buffer.alloc(inputLen, fs.readFileSync(__filename));
```

Comment:
Nit: any reason for using `fs.readFileSync(__filename)` instead of an arbitrary fill character?

Reply:
@lpinca Yes, exactly that. If you have other suggestions, I’ll take them too :)

Reply:
Nah, that's ok and I have no better ideas.